crates/node/src/catchup.rs

```diff
@@ -339,26 +338,23 @@ impl Node {
 
                 if !self
                     .ctx_manager
-                    .is_application_installed(&change.application_id)
+                    .is_application_installed(&change.application_id)?
                 {
+                    // note! for now, we assume all paths are urls
+                    // todo! for path sources, share the blob peer to peer
```

Comment:
highlighting for visibility

catchups in development mode will attempt to download the "file", which "just might work" locally

worth testing

and if not, well.. odds are, your other node over the local network is in dev mode too

eventually we'll change this to peer-to-peer blob sharing